docker-compose.yml

```diff
@@ -13,6 +13,8 @@ frontend:
 backend:
   image: codeforpoznan/volontulo
   # build: .
+  environment:
+    - SECRET_KEY=a63eb5ef-3b25-4595-846a-5d97d99486f0
```

Comment:
Can we make this coherent with the rest of file? we use format
```
POSTGRES_PASSWORD: volontulo
```
instead of
```
- SECRET_KEY=a63eb5ef-3b25-4595-846a-5d97d99486f0
```